src/args.rs

```diff
@@ -681,7 +687,12 @@ impl Args {
 
     /// Create a new recursive directory iterator at the path given.
     pub fn walker(&self, path: &Path) -> Result<walk::Iter> {
-        let wd = WalkDir::new(path).follow_links(self.follow);
+        let wd =
+            if self.maxdepth > 0 {
```

Comment:
I think we want to allow a value of `0` to mean "search current directory." In this case, `maxdepth` should be an `Option<usize>`, which docopt will handle correctly.